src/Resources/views/base_layout.html.twig

```diff
@@ -114,7 +120,7 @@ file that was distributed with this source code.
 
                                     {% if page is defined %}
                                         <li>
-                                            <a href="{{ sonata_admin.url('sonata.page.admin.page', 'compose', {'id': page.id}) }}">
+                                            <a href="{{ sonata_page_admin.generateUrl('compose', {'id': page.id}) }}">
```

Comment:
IMO, we should not hardcode the parameter name for the identifier nor how its value is obtained.
`AdminInterface` exposes the `getIdParameter()` and `id()` methods for these values.

Reply:
I agree, but trying to test this, something else has arisen, looks like this `page` variable is not an instance of let's say `App\Entity\Page`, it's an instance of `Sonata\PageBundle\Model\SnapshotPageProxy`, so it throws:

```
An exception has been thrown during the rendering of a template ("No entity manager defined for class Sonata\PageBundle\Model\SnapshotPageProxy").
```

Maybe related of https://github.com/sonata-project/SonataPageBundle/issues/513

Reply:
In that case, I think we should leave a comment exposing the problem and open a issue (if it isn't reported yet).

Reply:
ok, so I've ended up changing all the `objectUrl`|`generateObjectUrl` calls that failed by `generateUrl` and that should address the issues with that `SnapshotPageProxy` problem and added a comment.